lib/spell/serializer.ex

```diff
@@ -8,21 +8,21 @@ defmodule Spell.Serializer do
   alias Spell.Message
 
   @doc """
-  Returns the name of the serializer. This is used to construct the
-  sub-protocol name:
+  Returns the transport specific data of the serializer. This is used among
+  other things to construct the sub-protocol name:
 
       $transport.2.$serializer
 
   """
-  defcallback name :: String.t
+  defcallback transport_info(module :: atom) :: map
 
   @doc """
-  Set the pid which the transport should send received messages to.
+  Decodes a binary string encoded message
```

Comment:
This doc was confusing, I changed to something that make sense to me, but I can be missing something